frontend/src/components/Vote.js

```diff
@@ -133,39 +146,34 @@ export default function VoteDialog(props) {
                   }
                   label="Abstain"
                 />
-
               </FormGroup>
             </FormControl>
-            {
-              props.isAuthzMode && props.granters.length > 0 ?
-                <FormControl fullWidth
-                  sx={{
-                    mt: 1,
+            {props.isAuthzMode && props.granters.length > 0 ? (
+              <FormControl
+                fullWidth
+                sx={{
+                  mt: 1,
+                }}
+              >
+                <InputLabel id="granter-label">Granter Account</InputLabel>
+                <Select
+                  labelId="granter-label"
+                  id="granter-select"
+                  value={granter}
+                  label="Granter account"
+                  onChange={(e) => {
+                    setGranter(e.target.value);
                   }}
+                  size="small"
                 >
-                  <InputLabel id="granter-label">Granter Account</InputLabel>
-                  <Select
-                    labelId="granter-label"
-                    id="granter-select"
-                    value={granter}
-                    label="Granter account"
-                    onChange={(e) => {
-                      setGranter(e.target.value);
-                    }}
-                    size="small"
-                  >
-                    {
-                      props.granters.map((granter, index) => (
-
-                        <MenuItem id={index} value={granter}>{granter}</MenuItem>
-                      ))
-                    }
-                  </Select>
-
-                </FormControl>
-                :
-                null
-            }
+                  {props.granters.map((granter, index) => (
+                    <MenuItem id={index} value={granter}>
+                      {fetchName('cosmos1vv6hruquzpty4xpks9znkw8gys5x4nsnqw9f4k') || granter}
```

Comment:
If any default values want to use in the code, use them from the constants or env.